### PR TITLE
Final touch ups to athletes landing page

### PR DESCRIPTION
**Pull request by tanyaparker**
Final video and social media tags are updated [here](https://github.com/code-dot-org/code-dot-org/pull/12253/).

image

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/athletes.haml [hunk 1]
```diff
@@ -37,7 +37,7 @@ video_player: true
   - url = "https://www.youtube.com/watch?v=#{yt_code}"
   - facebook = {u: url}
   - twitter = {url: url, related: 'codeorg', text: "#{I18n.t(:sports_og_description)} @codeorg"}
-  =view :display_video_thumbnail, id: 'athletes_video', video_code: yt_code, caption: I18n.t(:video_1min), play_button: 'center', facebook: facebook, twitter: twitter, download_path: download_link
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/athletes.haml:40]: I know in that email thread Hadi wanted to shorten the caption to this but I think it's unnecessary to keep the length if we don't have a full caption. Makes more sense in context like the other video captions.

File: pegasus/cache/i18n/en-US.yml [hunk 2]
```diff
@@ -432,6 +432,11 @@
   video_title_obama: "President Obama on computer science"
   video_title_hoc_is_here: "Hour of Code is Here - Anybody Can Learn"
   video_title_anybody_can_learn: "Anybody Can Learn"
+  video_title_draymond_green: "Hour of Code with Draymond Green"
```
Comment (tanyaparker) [inline pegasus/cache/i18n/en-US.yml:454]: Added these strings to I18n gsheet

File: pegasus/sites.v3/code.org/public/athletes.haml (hunk 1 above)
Comment (caleybrock) [reply pegasus/sites.v3/code.org/public/athletes.haml:40]: I'll add back in the caption. Apparently we'll need the athlete's names.

**Comment by caleybrock** [comment]
looks good so far 👍

**Pull request merged**